### PR TITLE
Makes peaches used in crafting a lot more, grammer fixes and new icecreams and floats - Remakes charrie shakes into icecreams rather then drinks

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Needs New plant type: Peaches #9146

## About The Pull Request

I cant spell shit, whats new?
Adds more icecream/snowcones - grape, peach
Adds tons of peach crafting items, like pies, cakes and more!
Adds many new things like banana splits, cola float
Remakes Charrie shakes into crafting rather then mixing chems
Adds in a new book to give cooks the crafting knowing of these new and grand foods.
Adds in mustard to conidements
Adds in coco-banana
Adds in corn dog meal
More paper cups in the dinnerwaver vender thing

## Why It's Good For The Game

Cooking Simulator, and more adv icecreams that are cool!

## Changelog
:cl:
add: tons of peach themed items
spellcheck: caje
/:cl:

### CODE REVIEW

File: code/game/objects/items/granters.dm [hunk 1]
```diff
@@ -440,7 +440,15 @@
...
 	remarks = list("So that is how icing is made!", "Placing fruit on top? How simple...", "Huh layering cake seems harder then this...", "This book smells like candy", "A clown must have made this page, or they forgot to spell check it before printing...", "Wait, a way to cook slime to be safe?")
 
-//Later content when I have free time - Trilby Date:02-Aug-2019
+/obj/item/book/granter/crafting_recipe/coldcooking //IceCream
+	name = "Cooking with Ice"
+	desc = "A cook book that teaches you many old icecream treats."
+	crafting_recipe_types = list(/datum/crafting_recipe/food/banana_split, /datum/crafting_recipe/food/root_float, /datum/crafting_recipe/food/bluecharrie_float, /datum/crafting_recipe/food/charrie_float)
+	icon_state = "cooking_learing_ice"
+	oneuse = FALSE
+	remarks = list("Looks like these would sell much better in a plasma fire...", "Using glass bowls rather then cones?", "Mixing soda and ice-cream?", "Tall glasses with of liquids and solids...", "Just add a bit of icecream and charry on top?")
```
Comment (Thalpy) [inline code/game/objects/items/granters.dm]: Cherry / cherries not charry/charrie

File: code/modules/food_and_drinks/food/condiment.dm [hunk 2]
```diff
@@ -16,6 +16,7 @@
 	//Possible_states has the reagent id as key and a list of, in order, the icon_state, the name and the desc as values. Used in the on_reagent_change(changetype) to change names, descs and sprites.
 	var/list/possible_states = list(
 	 "ketchup" = list("ketchup", "ketchup bottle", "You feel more American already."),
+	 "mustard" = list("mustard", "mustard bottle", "A spice mixed with enzymes and water."),
```
Comment (Thalpy) [inline code/modules/food_and_drinks/food/condiment.dm:19]: I didn't know that the enzymes in ground mustard seeds turned the glucosinolates into isothiocyanate compounds with water. That's neat!

File: code/modules/food_and_drinks/food/snacks_frozen.dm [hunk 3]
```diff
@@ -28,6 +28,50 @@
...
+	bonus_reagents = list("nutriment" = 5, "vitamin" = 3)
+	list_reagents = list("nutriment" = 3, "banana" = 10, "vitamin" = 2)
+	filling_color = "#FFFACD"
+	tastes = list("ice cream" = 1, "banana" = 1, "charries" = 1)
+	foodtype = FRUIT | DAIRY | SUGAR
+
+/obj/item/reagent_containers/food/snacks/cola_float
+	name = "Root Beer Float"
+	trash = /obj/item/reagent_containers/food/drinks/drinkingglass
+	desc = "A glass filled with cream, soda and ice-cream with a charrie on top."
```
Comment (Thalpy) [inline code/modules/food_and_drinks/food/snacks_frozen.dm]: Cherry on top

File: code/modules/food_and_drinks/food/snacks_frozen.dm [hunk 4]
```diff
@@ -28,6 +28,50 @@
...
+	icon_state = "cola_float"
+	bonus_reagents = list("nutriment" = 3, "vitamin" = 1)
+	list_reagents = list("nutriment" = 3, "vitamin" = 2)
+	filling_color = "#FFFACD"
+	tastes = list("ice cream" = 1, "space coal" = 1, "charries" = 1)
```
Comment (Thalpy) [inline code/modules/food_and_drinks/food/snacks_frozen.dm]: Cherries

File: code/modules/food_and_drinks/food/snacks_pie.dm [hunk 5]
```diff
@@ -319,8 +319,16 @@
...
-	desc = "A strawberry.pie."
+	desc = "A strawberry pie."
 	icon_state = "strawberrypie"
 	bonus_reagents = list("nutriment" = 6, "vitamin" = 6)
 	tastes = list("strawberry" = 1, "pie" = 1)
 	foodtype = GRAIN | FRUIT | SUGAR
+
+/obj/item/reagent_containers/food/snacks/pie/peachpie
+	name = "peach pie"
+	desc = "A pei with peach filling."
```
Comment (Thalpy) [inline code/modules/food_and_drinks/food/snacks_pie.dm]: Pie

**Review by Thalpy — COMMENTED** [review]
Spellchecked for you.

**Review by nik707 — APPROVED** [review]

**Pull request merged**